### ISSUE TITLE
Mistake in documentation for sigma draws

### ISSUE THREAD

**Issue by EoghanONeill** [rsparapa/bnptools#5]
The documentation currently says for the function ``wbart`` that the output ``sigma`` is
```post burn in draws of sigma, length = ndpost.```

However, it actually includes burn-in draws. Also the item ``first.sigma`` does not exist.

**Comment by rsparapa** [comment]
Good catch.  Unfortunately, I'm sure that there are lots of typos.  We inherited the documentation from BayesTree.  But, over time, we have made changes that now clash.

**Comment by Gattocrucco** [comment]
I guess this can be closed now because it's been fixed by 6174a2e
